lib/puppet-lint/data.rb

```diff
@@ -81,7 +81,7 @@ def title_tokens
               }
               title_array_tokens = tokens[(array_start_idx + 1)..(token_idx - 2)]
               result += title_array_tokens.select { |token|
-                {:STRING => true, :NAME => true}.include? token.type
+                [:STRING, :NAME].include? token.type
```

Comment:
The reason I went with hashes here is because Array#include? is O(n) and Hash#include? is O(1). If we're going to clean this up, I'd like to see this become a Set instead of an Array so that we don't slow things down.